src/hooks/use-mint-caps.ts

```diff
@@ -3,15 +3,19 @@ import getEmilyLimits from "@/actions/get-emily-limits";
 import { useQuery } from "@tanstack/react-query";
 import { useCallback, useMemo } from "react";
 
+export function useEmilyLimits() {
+  return useQuery({
+    queryKey: ["deposit-max-fee"],
```

Comment:
nit: update the key? even if doesn't matter as we are using the same `useEmilyLimits` around